src/main/include/log4cxx/logger.h

```diff
@@ -2056,19 +2056,21 @@ Add a new logging event containing \c message to attached appender(s) if this lo
 /**
 Add a new logging event containing \c message to attached appender(s) if \c logger is enabled for <code>DEBUG</code> events.
 
-@param logger the logger that has the enabled status.
-@param message a valid r-value expression of an <code>operator<<(std::ostream&. ...)</code> overload.
-
+\usage
 ~~~{.cpp}
 LOG4CXX_DEBUG(m_log, "AddMesh:"
 	<< " name " << meshName
 	<< " type 0x" << std:: hex << traits.Type
```

Comment:
It's not something that you modified here, but won't the `std::hex` here also affect the later calls of `getCenter()` and `getHalfSize()`? I think the two examples(LOG4CXX_DEBUG and LOG4CXX_DEBUG_FMT) are not equivalent in that case.

Reply:
> the later calls of getCenter() and getHalfSize()

These actually return 3d float vectors which are handled by a `operator<<(std::ostream& ` overload. Not obvious I know, but I wanted to use a real example.